### PR TITLE
[GeoMechanicsApplication] Add cross-sectional area to the thermal line element

### LINKED ISSUE #12346: [GeoMechanicsApplication] Add cross-sectional area to the thermal line element
### Description
The line element has to be extended to include the sectional area. All terms in the heat equation has to be multiplied with the sectional area. In principle, it does not make change in the results for an isolated 1D line element. The effect will appear when the 1D line element gets combined with 2D element, for example, when a well or pipe inside the soil gets simulated. The matrices of this 1D element will be superposined with the 2D elements in the main matrix.

### PR DESCRIPTION
**Pull request by mnabideltares**
**📝 Description**
The line element has to be extended to include the sectional area. All terms in the heat equation has to be multiplied with the sectional area. In principle, it does not make change in the results for an isolated 1D line element. The effect will appear when the 1D line element gets combined with 2D element, for example, when a well or pipe inside the soil gets simulated. The matrices of this 1D element will be superposined with the 2D elements in the main matrix.

**🆕 Changelog**
- Added cross-section area to the thermal line element
- Added "CROSS_AREA" to MaterialParameter.json
- Removed duplicated files of MaterialParameter.json by putting them in a common dairectory

### CODE REVIEW

**Review by WPK4FEM — APPROVED** [review]
Hi Mohammed,
Thank you, this removes a good amount of duplication by reuse of the MaterialProperties.
In this way all existing tests keep giving the same answer. It would be good if a future test that combines line and volume (or plane) elements contains a CROSS_AREA that is not 1.0

File: applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h [hunk 1]
```diff
@@ -242,13 +242,17 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
...
         const auto& r_integration_points = GetGeometry().IntegrationPoints(GetIntegrationMethod());
 
         auto result = Vector{r_integration_points.size()};
         for (unsigned int integration_point_index = 0;
              integration_point_index < r_integration_points.size(); ++integration_point_index) {
             result[integration_point_index] = r_integration_points[integration_point_index].Weight() *
                                               rDetJContainer[integration_point_index];
+            if (GetGeometry().LocalSpaceDimension() == 1) {
+                result[integration_point_index] *= r_properties[CROSS_AREA];
+            }
```
Comment (rfaasse) [inline applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h:255]: Would this be the same for the Pw variant? If yes, do you think it's best to fix that in the same PR, or a different one?

**Review by rfaasse — APPROVED** [review]
Looks good! If this same issue is there for the Pw line elements, would you like to do that as part of this PR, or a separate one? Leaving that up to you, looks good to go from my side

File: applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h (hunk 1 above)
Comment (mnabideltares) [reply applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h:255]: @rfaasse Thank you for the review.
Indeed for Pw a similar feature needs to be added. However, currently I am focousing on the heat only to reach the sprint goal. Will do it for Pw is a seperate PR

**Pull request merged**